camelot/view/controls/tableview.py

```diff
@@ -458,6 +474,23 @@ def __init__(self, gui_context, parent):
         self.setLayout(layout)
         self.setSizePolicy(QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Fixed)
 
+    @QtCore.qt_slot()
+    def close_clicked(self):
+        app = QtWidgets.QApplication.instance()
```

Comment:
deze 'zoek' is in feite niet nodig hoor, wat je best kan doen denk ik is een signal definieren op de tableview, en dit signal emitten, in de tab widget 'connect' je op dit signal bij het toevoegen vd tab.

in een slot kan je steeds self.sender() callen, om te weten van waar het signal komt, dwz dat je in de slot vd tab view weet welke tab heeft gevraagd om afgesloten te worden.

Reply:
Dit is inderdaad een betere oplossing. Ik heb het signal toegevoegd aan de AbstractView klasse.